docs/usage.adoc

```diff
@@ -64,22 +64,34 @@ Environment variables allow you to redefine the default settings:
 
 * `EMERALD_HOST` - RPC server listen host
 * `EMERALD_PORT` - RPC server listen port
-* `EMERALD_CHAIN` - chain name (`mainnet` | `morden`), has a higher priority relative to `EMERALD_CHAIN_ID`
+* `EMERALD_CHAIN` - chain name (see Table.1, `CLI argument` column), has a higher priority relative to `EMERALD_CHAIN_ID`
 * `EMERALD_CHAIN_ID` - chain id number, has a lower priority relative to `EMERALD_CHAIN`
 * `EMERALD_GAS` - maximum gas limit to use by transaction
 * `EMERALD_GAS_PRICE` - gas cost to use by transaction (in Gwei)
 * `EMERALD_SECURITY_LEVEL` - security level (`normal` | `high` | `ultra`)
 * `EMERALD_NODE` - url to upstream node. Used for sign and send of transactions
 
+| Chain id  | Description                       | CLI argument   |
+|-----------|-----------------------------------|----------------|
+| 1         | Ethereum mainnet                  | eth            |
+| 2         | Morden (disused), Expanse mainnet | morden         |
+| 3         | Ropsten                           | ropsten        |
+| 4         | Rinkeby                           | rinkeby        |
+| 30        | Rootstock mainnet                 | rootstock-main |
+| 31        | Rootstock testnet                 | rootstock-test |
+| 42        | Kovan                             | kovan          |
+| 61        | Ethereum Classic mainnet          | etc-main       |
+| 62        | Ethereum Classic testnet          | etc-test       |
```

Comment:
geth uses morden, why vault has own testnet instead?

Reply:
eth morden chain id is 2, etc morden - 62

Reply:
should (or is?) CLI argument `etc` be synonym for `etc-main`? 

like `eth` is implicit for `eth-main`?

Reply:
and let's use `mainet`/`testnet` instead of `main`/`test`, to avoid confusion. I mean `etc-mainnet` (as a synonym to simple `mainnet`), not `etc-main` which will be a totally new name